fitbenchmarking/core/results_output.py

```diff
@@ -80,10 +80,10 @@ def generate_tables(results_per_test, minimizers,
                     for name in table_suffix]
     results_dict, html_links = create_results_dict(results_per_test,
                                                    linked_problems)
-    preproccess_data(results_dict)
+    error_options = preproccess_data(results_dict)
```

Comment:
`error_options` is always the same dict, so we don't need to percolate it through the code -- I think just setting it at the point we need would do the job.

Reply:
Done